src/inventory_ui.cpp

```diff
@@ -3092,6 +3095,8 @@ drop_locations pickup_selector::execute()
             return drop_locations();
         } else if( input.action == "INVENTORY_FILTER" ) {
             query_set_filter();
+        } else if( input.action == "RESET_FILTER" ) {
+            set_filter( "" );
```

Comment:
I don't think the explicit handling is needed here if you already do it in `inventory_selector::on_input`?

Reply:
Good point, thanks. The branch immediately above is also unnecessary, but I've left that alone for now.

Reply:
Yeah, both `pickup_selector::execute` and `inventory_drop_selector::execute` could use some more refactoring and generalization. But that's something for its own PR.